nns/recordtype.go

```diff
@@ -3,6 +3,9 @@ package nns
 // RecordType is domain name service record types.
 type RecordType byte
 
+// WARNING!
```

Comment:
can we move that into `const`? practice says devs look max 2 lines above when fixing/extending smth

Reply:
I've specifically tried to avoid having it in godoc, since it's only relevant for developers. The file is rather small and not changed often.